presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestingPrestoClient.java

```diff
@@ -143,6 +143,7 @@ private static ClientSession toClientSession(Session session, URI server, boolea
                 session.getLocale(),
                 properties.build(),
                 session.getPreparedStatements(),
+                session.getIdentity().getRoles(),
```

Comment:
It is not. But the place where we take roles did. So, now we retrieve them from identity, instead of directly from Session.